README.md

```diff
@@ -46,3 +46,12 @@ $# ./bin/spark-submit \
 Notice how we're referencing spark-master not by IP, but by k8s Service name
 "spark-master.default.svc.cluster", which we've previously deployed. This way we don't even need
 to manually lookup the spark-master's IP, but rather leave it to the Kubernetes to do this for us.
+
+## License
+Unless specifically noted, all parts of this project are licensed under the [Apache 2.0 license](LICENSE).
+
+## Acknowledgements
+This project has been conducted within the RIA [MIKELANGELO
```

Comment:
There is no need to add RIA acronym here. This is internal EC naming which may change in the future.

Reply:
This text was copied from somewhere initially as all `mikelangelo-project` repos have the same acronym there. Should we remove them in other places as well?

Reply:
If it's not too much work, I think this would be very good. Where's this coming from? You remember IP and STREP  projects? These are now RIA and IA. When this changes, people will look strangely at the "RIA" acronym AND the whole code will look old and obsolete.

Reply:
Done, I've simply removed the "Acknowledgements" section.

Reply:
I don't want to point any fingers 😄 so I'll just say that it was copied from some place I do not remember.

@miha-plesko remove this here now and we'll request a change on other repositories